xbmc/utils/MemUtils.h

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <cstdint>
+#include <memory>
+
+namespace KODI
+{
+namespace MEMORY
+{
+  struct MemoryStatus
+  {
+    unsigned int length;
```

Comment:
Uh what is this used for? Why do we need it?

Reply:
I just kept the same windows idioms. I can remove it though now that you pointed it out :)

https://docs.microsoft.com/en-gb/windows/desktop/api/sysinfoapi/nf-sysinfoapi-globalmemorystatusex

Reply:
Yes, please remove :3